queries.sql

```diff
@@ -1,17 +1,70 @@
 /*Queries that provide answers to the questions from all projects.*/
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
NOTED THANKS